conans/client/generators/cmake_find_package_multi.py

```diff
@@ -168,15 +169,15 @@ class CMakeFindPackageMultiGenerator(CMakeFindPackageGenerator):
 
         # Load the debug and release library finders
         get_filename_component(_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
-        file(GLOB CONFIG_FILES "${_DIR}/{{ pkg_name }}Target-*.cmake")
+        file(GLOB CONFIG_FILES "${_DIR}/{{ pkg_filename }}Target-*.cmake")
 
         foreach(f ${CONFIG_FILES})
             include(${f})
         endforeach()
 
-        if({{ pkg_name }}_FIND_COMPONENTS)
-            foreach(_FIND_COMPONENT {{ '${'+pkg_name+'_FIND_COMPONENTS}' }})
-                list(FIND {{ pkg_name }}_COMPONENTS_{{ build_type }} "{{ pkg_name }}::${_FIND_COMPONENT}" _index)
+        if({{ pkg_filename }}_FIND_COMPONENTS)
+            foreach(_FIND_COMPONENT {{ '${'+pkg_filename+'_FIND_COMPONENTS}' }})
+                list(FIND {{ pkg_filename }}_COMPONENTS_{{ build_type }} "{{ pkg_name }}::${_FIND_COMPONENT}" _index)
```

Comment:
please keep the same variables names, this change seems unrelated to the filename implementation